walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -4,68 +4,105 @@
     <meta charset="UTF-8">
     <title>All About Jalyn 🐈</title>
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
   </head>
   <body>
-<div class="navigation_bar">
-    
+    	<div id="navigation-bar">
+        	<span class="nav nav-btn"> <a class="nav" href= "index.html" style="background-color: white">Home</a> </span>
+        	<span class="nav nav-btn"> <a class="nav" href="Resume_CS.pdf" style="background-color: white">Resume</a> </span>
+        	<span class="nav nav-btn"> <a class="nav" href="/socials.html" style="background-color: white">Socials</a> </span>
+    	</div>
 
-	<h3><a class="nav" href="/random.html">Link One</a></h3>
+        <div class="full-page">
+            <div id="first-page">
+                <h1>The One Where Jalyn Makes A Portfolio...</h1>
+                <h2> We're about to get real close</h2>
+                <img src="/images/hello.jpeg" id="top-img">
+            </div>
+        </div>
+
+
+        <div id="second-page" class= "full-page">
+            <ul class="caption-style-1">
+                <li> <img src="/images/new-zealand.jpg" alt="">
```

Comment:
format li element to be on separate line to img element

Reply:
done!